tests/countries/test_palestine.py

```
@@ -0,0 +1,438 @@
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import CATHOLIC, ORTHODOX
+from holidays.countries.palestine import Palestine, PS, PSE
+from tests.common import CommonTests
+
+
+class TestPalestine(CommonTests, TestCase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)